scripts/zowe-support.template.sh

```diff
@@ -40,7 +40,10 @@ if [[ -z "${ZOWE_PREFIX}" ]];then
     echo "The ZOWE_PREFIX environment variable wasn't properly populated during install. Exiting."
     exit
 fi
-
+if [[ -z "${ZOWE_INSTANCE}" ]];then
```

Comment:
ZOWE_INSTANCE doesn't seem to be defined in this file, so this if block will always trigger. I also don't see the need for it